core/api-changes.json

```diff
@@ -2,6 +2,15 @@
   "4.3.0": {
     "revapi": {
       "ignore": [
+        {
+          "code": "java.method.addedToInterface",
```

Comment:
this method should have a default implementation, otherwise, artifacts with implementations of this interface will break